docs/reference/buildx_history_export.md

```diff
@@ -15,3 +15,31 @@ Export a build into Docker Desktop bundle
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+Export one or more build records to `.dockerbuild` archive files. These archives
+contain metadata, logs, and build outputs, and can be imported into Docker
+Desktop or shared across environments.
+
+## Examples
+
+### Export a single build to a custom file
+
+```console
+docker buildx history export mybuild --output mybuild.dockerbuild
+```
+
+### Export multiple builds to individual `.dockerbuild` files
+
+This example writes `mybuild.dockerbuild` and `backend-build.dockerbuild` to
+the current directory:
+
+```console
+docker buildx history export mybuild backend-build
```

Comment:
The last two examples would fail in terminal. Supported variants are `-o/--output filename` or `docker buildx history export [ref] > mybuild.dockerbuild`